ward/testing.py

```diff
@@ -604,30 +604,6 @@ def _resolve_single_arg(
                 fixture.key, fixture.scope, self.test.scope_key_from(fixture.scope)
             )
 
-        has_deps = len(fixture.deps()) > 0
```

Comment:
@AABur - This looks like a nice find and good refactor 👏 At first I thought it was a mistake, but it looks like this almost identical code directly below it covers the `has_deps == False` case anyway.

Reply:
Exactly right!